lib/audited/auditor.rb

```diff
@@ -204,7 +200,7 @@ def audited_changes
         if audited_options[:only].present?
           all_changes.slice(*audited_columns)
         else
-          all_changes.except(*non_audited_columns)
+          all_changes.except(*self.class.non_audited_columns)
```

Comment:
Why is this change needed? why does it matter if we have an instance method that calls the class method just for convenience?

Reply:
- having an instance method for a class variable makes it look like it's changeable/configured on the instance, so extra mental overhead and some users might think it's ok to do `instance.non_audited_columns << :foo`
 - it's not much convenience if it's only needed once
 - for symmetry also removed `def audited_columns `